### PR TITLE
PIM-9406: Merge 3.0 into 3.2

### PR DESCRIPTION
**Pull request by anaisbaunelemaire**
**Description (for Contributor and Core Developer)**

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | Todo
| Added legacy Behats               | Todo
| Added acceptance tests            | Todo
| Added integration tests           | Todo
| Changelog updated                 | Todo
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | Todo
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -65,6 +65,7 @@
     "@types/backbone": "^1.3.42",
     "@types/node": "11.9.5",
     "@types/underscore": "~1.9.4",
+    "@types/underscore": "~1.9.4",
```
Comment (rouliane) [inline package.json:68]: duplicated line, to remove

File: CHANGELOG-3.0.md [hunk 2]
```diff
@@ -1,5 +1,19 @@
...
+
+## Improvements
+
+- PIM-9383: Fix underscore JS dependency
+
+# 3.0.82 (2020-07-23)
+
+## Bug fixes
+
+- PIM-9282: Make calling attribute options via API case insensitive
```
Comment (ValentinMumble) [inline CHANGELOG-3.0.md:13]: The pullup doesn't seem to include changes related to this fix 🤔

Comment (anaisbaunelemaire) [reply CHANGELOG-3.0.md:13]: So strange, indeed... I'll take a look at this.

Comment (anaisbaunelemaire) [reply CHANGELOG-3.0.md:13]: @ValentinMumble It seems that JM had made the fix in 3.0 AND in 3.2.
Hence the absence of fix in the 3.0 merge => 3.2.

**Pull request merged**